src/NuGet.Core/NuGet.CommandLine.XPlat/Utility/AddPackageCommandUtility.cs

```diff
@@ -88,9 +88,10 @@ public static async Task<NuGetVersion> GetLatestVersionFromSourceAsync(PackageSo
         /// Returns the PackageSource with its credentials if available
         /// </summary>
         /// <param name="requestedSources">Sources to match</param>
+        /// <param name="additionalSources">Additional user supplied source feeds as argument</param>
         /// <param name="configFilePaths">Config to use for credentials</param>
         /// <returns>Return a list of package sources</returns>
-        public static List<PackageSource> EvaluateSources(IList<PackageSource> requestedSources, IList<string> configFilePaths)
+        public static List<PackageSource> EvaluateSources(IList<PackageSource> requestedSources, string[] additionalSources, IList<string> configFilePaths)
```

Comment:
you added this new method, but you're not really testing the new capability.

Reply:
My 2 unit tests already cover this one.